src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/AbstractRecordCollection.java

```diff
@@ -89,8 +112,11 @@ public final int size() {
      * Writes the records to file.
      */
     public void write(final File outputFile) {
-        try (final RecordCollectionWriter writer = new RecordCollectionWriter(outputFile)) {
-            writer.writeAllRecords(records);
+        try (final FileWriter writer = new FileWriter(outputFile, true)) {
```

Comment:
This needs to be modified to first overwrite existing files with the header before appending the records.  Currently, if the file exists, this will just append everything.